wsd/DocumentBroker.cpp

```diff
@@ -2980,12 +3000,7 @@ void DocumentBroker::handleUploadToStorageSuccessful(const StorageBase::UploadRe
 
 void DocumentBroker::handleUploadToStorageResponse(const StorageBase::UploadResult& uploadResult)
 {
-    if (!_uploadRequest)
-    {
-        // We shouldn't get here if there is no active upload request.
-        LOG_ERR("No active upload request while handling upload result.");
-        return;
-    }
+    assert(_uploadRequest && "Expected to have a valid UploadRequest instance");
```

Comment:
Hmm - we change behavior here - is that deliberate ? an assert is compiled out, but the if() is there at run-time and might protect us; can we do both ?